sentry-android-core/build.gradle.kts

```diff
@@ -64,6 +66,27 @@ android {
     }
 }
 
+tasks {
```

Comment:
would be possible to reuse the tasks block across all the Android repos?

Reply:
I did not figure out how. Contributions welcome :-)